onnxruntime/core/framework/instant_op.cc

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/framework/data_types.h"
```

Comment:
General nit comment: The usage of "Instant" in a number of places isn't easy to associate with this feature (standalone graphless kernels). Don't know what would be an appropriate buzz word though...

Reply:
Yes - not prefer "eager" either, any better ideas than "instant" please suggest.